data/transactions/logic/opcodes.go

```diff
@@ -154,16 +154,16 @@ var OpSpecs = []OpSpec{
 	{0x60, "balance", opBalance, asmDefault, disDefault, oneInt, oneInt, 2, runModeApplication, opSizeDefault},
 	{0x61, "app_opted_in", opAppCheckOptedIn, asmDefault, disDefault, twoInts, oneInt, 2, runModeApplication, opSizeDefault},
 	{0x62, "app_local_get", opAppGetLocalState, asmDefault, disDefault, oneInt.plus(oneBytes), oneAny, 2, runModeApplication, opSizeDefault},
-	{0x63, "app_local_get_ex", opAppGetLocalStateEx, asmDefault, disDefault, twoInts.plus(oneBytes), oneInt.plus(oneAny), 2, runModeApplication, opSizeDefault},
+	{0x63, "app_local_get_ex", opAppGetLocalStateEx, asmDefault, disDefault, twoInts.plus(oneBytes), oneAny.plus(oneInt), 2, runModeApplication, opSizeDefault},
```

Comment:
The problem with this change is that this changes the *spec*
The implementation in `opAppGetLocalStateEx()` isn't the spec.
`(cd data/transactions/logic && rm -f TEAL_opcodes.md; make)`
Then you see it changes the opcode spec.

Reply:
I guess the spec can be simply extended to specify pushing order. Currently it is not clear in the spec doc with direction the stack grows. I updated the spec and the spec generator.